cyhy/db/database.py

```diff
@@ -1213,36 +1213,67 @@ def get_descendants(org_info, current_org, stakeholders_only, include_retired):
 
     def get_owner_to_type_dict(self, stakeholders_only=False, include_retired=False):
         """returns a dict of owner_id:type. "stakeholders_only" parameter eliminates non-stakeholders from the dict."""
-        types = dict()
+        types = defaultdict(lambda: list())
         for agency_type in AGENCY_TYPE:
             all_agency_type_descendants = self.get_all_descendants(
                 agency_type, include_retired=include_retired
             )
             for org in self.find({"_id": {"$in": all_agency_type_descendants}}):
                 if stakeholders_only:
                     if org["stakeholder"]:
-                        types[org["_id"]] = agency_type
+                        types[org["_id"]].append(agency_type)
                 else:
-                    types[org["_id"]] = agency_type
+                    types[org["_id"]].append(agency_type)
```

Comment:
Unnecessary, but could this DRY out to:
```python
if not stakeholders_only or org["stakeholder"]:
    types[org["_id"]].append(agency_type)
```
?

Reply:
Good idea- thanks for mentioning it! Done in https://github.com/cisagov/cyhy-core/pull/47/commits/c74f424416bbeb5ad447fb065b848ed1d0b4fd41.